code/modules/hydroponics/grown/replicapod.dm

```diff
@@ -22,6 +22,7 @@
 	var/list/features = null
 	var/factions = null
 	var/list/quirks = null
+	var/sampleDNA = null
```

Comment:
can you delete this garbo so it's all

`var/XYZ` instead of `var/XYZ = null`

Reply:
yeah that's reasonable